### PR TITLE
Perfmon counters added

### PR DESCRIPTION
**Pull request by datacore-skumar**
- Three perfmon counter set added:
1. OpenZFS Zpool
2. OpenZFS Vdev
3. OpenZFS Cache

### CODE REVIEW

File: module/os/windows/zfs/zfs_ioctl_os.c [hunk 1]
```diff
@@ -53,6 +53,11 @@
 #include <zfs_gitrev.h>
 
 #include <Wdmsec.h>
+#include <sys/spa_impl.h>
+#include <sys/zfs_ioctl.h>
+#include "../OpenZFS_perf.h"
+#include "../OpenZFS_counters.h"
```
Comment (lundman) [inline module/os/windows/zfs/zfs_ioctl_os.c:59]: I'm guilty of using this from time to time, but out of curiosity, where are these headers? Is it a place we should add to build system, if they don't live in os/windows/ ?

**Review by lundman — APPROVED** [review]
This looks great, thanks for revisting it.

File: module/os/windows/zfs/zfs_ioctl_os.c (hunk 1 above)
Comment (datacore-skumar) [reply module/os/windows/zfs/zfs_ioctl_os.c:59]: module/os/windows/CMakeLists.txt
```
execute_process(COMMAND  "C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0\\x86\\ctrpp.exe"
  -o ${CMAKE_CURRENT_SOURCE_DIR}/OpenZFS_perf.h
  -rc ${CMAKE_CURRENT_SOURCE_DIR}/OpenZFS_perf.rc
  -ch ${CMAKE_CURRENT_SOURCE_DIR}/OpenZFS_counters.h
 ${CMAKE_CURRENT_SOURCE_DIR}/OpenZFS.man )
```
Headers will be created on every build at os/windows/ .

Comment (lundman) [reply module/os/windows/zfs/zfs_ioctl_os.c:59]: Ah generated, gotcha!

Comment (datacore-skumar) [reply module/os/windows/zfs/zfs_ioctl_os.c:59]: `execute_process(COMMAND  C:\\Program Files (x86)\\Windows Kits\\10\\bin\\10.0.19041.0\\x86\\ctrpp.exe)`
changed static path to dynamic : `execute_process(COMMAND  "${WDK_ROOT}/bin/${WDK_VERSION}/x86/ctrpp.exe")`

**Pull request merged**